tiatoolbox/utils/misc.py

```diff
@@ -755,6 +755,30 @@ def unzip_data(
         Path.unlink(zip_path)
 
 
+try:
```

Comment:
if this is only required for tests, then why not move this to conftest? I am a bit concerned about adding this to `misc.py`